src/main/java/de/torui/coflsky/WSCommandHandler.java

```diff
@@ -41,13 +44,26 @@ public static boolean HandleCommand(JsonStringCommand cmd, Entity sender) {
 		case ChatMessage:
 			ChatMessage(cmd.GetAs(new TypeToken<ChatMessageData[]>() {}));
 			break;
+		case Flip:
+			Flip(cmd.GetAs(new TypeToken<FlipData>() {}));
 		default:
 			break;
 		}
 
 		return true;
 	}
 
+	private static void Flip(Command<FlipData> cmd) {
+		//handle chat message
+		ChatMessageData[] messages = cmd.getData().Messages;
+		Command<ChatMessageData[]> showCmd = new Command<ChatMessageData[]>(CommandType.ChatMessage, messages);
+		ChatMessage(showCmd);
+		flipHandler.fds.Insert(new de.torui.coflsky.FlipHandler.Flip(cmd.getData().Id, cmd.getData().Worth));
```

Comment:
When holding the key down there is a noticeable delay between receiving flips (they even show up in chat) and the auction opening.
I am pretty sure that it is longer than my ping.
Could the opening be triggered in this method to optimise for opening auctions faster as it is very important